HW7_MVC/MiliFileManager/MliFileManager/ReportService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.IO;
+using TemplateEngine.Docx;
+
+namespace MiLiFileManager
+{
+    public sealed class ReportService
+    {
+        private const int ByteToGb = 1073741824;
+
+        private const string Template = @"Template/template_diskInfo.docx";
+
+        public static bool GenerateReport(string label, string output = "")
+        {
+            DriveInfo [] drives = DriveInfo.GetDrives();
+
+            DriveInfo driveInfo = null;
+            
+            foreach (var drive in drives)
+            {
+                var driveLabel = drive.Name.Substring(0, 1).ToLower();
+
+                if (label == driveLabel)
+                {
+                    driveInfo = drive;
+                }
+            }
+
+            if (driveInfo == null)
+            {
+                return false;
+            }
+
+            var pathDir = ConfigPath.GetPathFileConfig("");
+
+            var dateStampReport = $"{DateTime.Now.Second}.{DateTime.Now.Millisecond}";
```

Comment:
```suggestion
            var dateTimeNow = DateTime.Now;
            var dateStampReport = $"{dateTimeNow.Second}.{dateTimeNow.Millisecond}";
```

Избавит от повторного вычисления даты при каждом вызове